lang/c++/meson.build

```diff
@@ -0,0 +1,150 @@
+project('avro-cpp', 'cpp', version: '1.0', 
```

Comment:
I think including this without any build tooling to ensure it remains up to date is a really bad idea. It will immediately become out of date when someone next updates the CMakeLists file...

Reply:
If adding said tooling isn't trivial, or maybe even if it is, splitting into a second PR to make it easier to merge isn't a bad idea

Reply:
Sorry. I removed it. I didn't know PR would follow so to speak further commit.